### PR TITLE
Added el to Login Page (h3 Login selector), link to NavBar for Proble…

### PR DESCRIPTION
**Pull request by JuliaEde**
…ms.page.js and finished E2E test for navBarNavigationE2E.spec.js

### CODE REVIEW

File: test/ui/specs/navBarNavigationE2E.spec.js [hunk 1]
```diff
@@ -1,26 +1,60 @@
...
+    const email = `TestUser${Date.now()}@gmail.com`;
+    const password = 'superUser123!';
+    let userLoginRes;
+
     before('Create New User', async () => {
-        let email = `${Date.now()}@gmail.com`;
-        let password = 'superUser123!';
-        await createAndLoginAPI(email, password);
+        userLoginRes = await createAndLoginAPI(email, password);
+        console.log(userLoginRes);
```
Comment (yuliahar) [inline test/ui/specs/navBarNavigationE2E.spec.js:17]: Please delete console.log(). It was used for debugging.

File: test/ui/specs/navBarNavigationE2E.spec.js [hunk 2]
```diff
@@ -1,26 +1,60 @@
...
+    });
+
+    it('Should Navigate to Publications Page', async () => {
+        await Profile.navBar.openPublications();
+        await expect(PublicationsPage.navBar.pageTitle).toHaveText('publications');
+    });
+
+    it('Should Logout', async () => {
+        await PublicationsPage.navBar.logout();
+        await expect(LoginPage.pageTitle).toHaveText('Login');
```
Comment (yuliahar) [inline test/ui/specs/navBarNavigationE2E.spec.js:52]: Nice!

File: test/ui/specs/navBarNavigationE2E.spec.js [hunk 3]
```diff
@@ -1,26 +1,60 @@
...
+    });
+
+    after('Delete the User', async () => {
+        const userDeleteRes = await deleteUser({userId: userLoginRes.userID, accessToken: userLoginRes.accessToken});
+        console.log(userDeleteRes);
```
Comment (yuliahar) [inline test/ui/specs/navBarNavigationE2E.spec.js:57]: Please delete console.log

File: test/ui/specs/navBarNavigationE2E.spec.js [hunk 4]
```diff
@@ -1,26 +1,60 @@
...
+
     before('Create New User', async () => {
-        let email = `${Date.now()}@gmail.com`;
-        let password = 'superUser123!';
-        await createAndLoginAPI(email, password);
+        userLoginRes = await createAndLoginAPI(email, password);
+        console.log(userLoginRes);
+    });
+
+    it('Login', async () => {
```
Comment (yuliahar) [inline test/ui/specs/navBarNavigationE2E.spec.js:20]: Please move login to before(). This test suite checks navigation, not login.

**Review by yuliahar — CHANGES_REQUESTED** [review]

File: test/ui/specs/navBarNavigationE2E.spec.js (hunk 1 above)
Comment (JuliaEde) [reply test/ui/specs/navBarNavigationE2E.spec.js:17]: Done, deleted

File: test/ui/specs/navBarNavigationE2E.spec.js (hunk 4 above)
Comment (JuliaEde) [reply test/ui/specs/navBarNavigationE2E.spec.js:20]: Should I remove expect from before then?

File: test/ui/specs/navBarNavigationE2E.spec.js (hunk 3 above)
Comment (JuliaEde) [reply test/ui/specs/navBarNavigationE2E.spec.js:57]: Done. deleted

File: test/ui/specs/navBarNavigationE2E.spec.js (hunk 4 above)
Comment (yuliahar) [reply test/ui/specs/navBarNavigationE2E.spec.js]: yep

Comment (JuliaEde) [reply test/ui/specs/navBarNavigationE2E.spec.js]: Ok, thank you for the suggestion

Comment (yuliahar) [reply test/ui/specs/navBarNavigationE2E.spec.js]: 🤝

**Review by yuliahar — APPROVED** [review]

**Pull request merged**